admin/invoice.php

```diff
@@ -459,7 +460,12 @@ function couponpopupWindow(url) { /* just a stub for coupon output that might fi
           }
       ?>
     </div>
-
+<?php
+} else {
+    // no information for invoice
+    echo INVOICE_ERROR_ORDER_DOES_NOT_EXIST . $oID;
```

Comment:
Please handle the way admin/orders.php does (line 70-72) - add error to messageStack, redirect page.